lib/routes/recovery-codes.js

```diff
@@ -128,7 +128,7 @@ module.exports = (log, db, config, customs, mailer) => {
               if (remaining === 0) {
```

Comment:
nit: I find the name `remaining` a bit vague. `remainingRecoveryCodes` would be more clear.